cmd/finch/version_test.go

```diff
@@ -32,17 +35,22 @@ func captureStdout(t *testing.T, f func()) string {
 func TestVersionCommand(t *testing.T) {
 	t.Parallel()
 
-	cmd := newVersionCommand()
+	ctrl := gomock.NewController(t)
+	logger := mocks.NewLogger(ctrl)
+	lcc := mocks.NewLimaCmdCreator(ctrl)
+	cmd := newVersionCommand(lcc, logger)
 	assert.Equal(t, cmd.Name(), "version")
 }
 
-//nolint:paralleltest // TODO: Add t.Parallel after using dependency injection to pass a io.Writer to newVersionCommand.
-func TestVersionAction(t *testing.T) {
-	mockCmd := newVersionCommand()
-	versionActionFunc := func() {
-		require.NoError(t, versionAction(mockCmd, []string{}))
-	}
-
-	output := captureStdout(t, versionActionFunc)
-	assert.Contains(t, output, finchVersion)
-}
+//TODO: Ang's review
```

Comment:
For the version info related to nerdctl, we can mock the result from `CreateWithoutStdio`, an example to follow:
https://github.com/runfinch/finch/blob/main/cmd/finch/virtual_machine_init_test.go#L239
from line 239 to line 241.